lib/brcobranca/version.rb

```diff
@@ -1,5 +1,5 @@
 # -*- encoding: utf-8 -*-
 
 module Brcobranca
-  VERSION = '6.1.7'
+  VERSION = '6.2.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.